pkg/kubelet/kubelet_network_others.go

```diff
@@ -0,0 +1,22 @@
+// +build !linux
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+// Do nothing.
+func (kl *Kubelet) syncNetworkUtil() {}
```

Comment:
what about naming as `pkg/kubelet/kubelet_network_unsupported.go`

Reply:
it's same but `_others` is preferred because `_linux.go` is used for Linux builds.